nursery/set-graphical-window-text.yml

```diff
@@ -0,0 +1,9 @@
+rule:
+  meta:
+    name: set console window title
+    namespace: host-interaction/gui/console/set-text
```

Comment:
since there aren't tons of sub-namespaces here, lets use `host-interaction/gui/console`